### PR TITLE
Issue #4954 - Expose header size/length from HttpParser

### PR DESCRIPTION
**Pull request by joakime**
- Minimal new API for exposing byte counts to HttpChannel.Listener users

Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

### CODE REVIEW

File: jetty-http/src/main/java/org/eclipse/jetty/http/HttpParser.java [hunk 1]
```diff
@@ -365,6 +365,11 @@ public long getContentRead()
         return _contentPosition;
     }
 
+    public long getHeaderLength()
```
Comment (gregw) [inline jetty-http/src/main/java/org/eclipse/jetty/http/HttpParser.java]: `_headerBytes` is an int and it make no sense to every have header size that needs a long.

**Review by gregw — CHANGES_REQUESTED** [review]

File: jetty-http/src/main/java/org/eclipse/jetty/http/HttpParser.java (hunk 1 above)
Comment (joakime) [reply jetty-http/src/main/java/org/eclipse/jetty/http/HttpParser.java]: Yeah, I see that now.
Guess I just institutionally made it a long to match the `.getContentLength()` return value.
I'll change it.

Comment (joakime) [reply jetty-http/src/main/java/org/eclipse/jetty/http/HttpParser.java]: Changed.

**Review by gregw — APPROVED** [review]

**Comment by joakime** [comment]
Replaces PR #4955

**Pull request merged**